packages/components/src/components/block-group/block-group.tsx

```diff
@@ -163,7 +163,7 @@ export class BlockGroup extends LitElement implements InteractiveComponent, Sort
   /**
    * Fires when a user attempts to move an element using the sort menu and 'canPut' or 'canPull' returns falsy.
    *
-   * @deprecated No longer necessary.
+   * @deprecated in v3.3.0, removal target v6.0 - No longer necessary.
```

Comment:
Same as above with consistency considerations:
```suggestion
   * @deprecated in v3.3.0, removal target v6.0.0 - No longer necessary.
```